CHANGELOG_DEV.md

```diff
@@ -0,0 +1,20 @@
+## 0.85.0 - 2020-04-02
```

Comment:
I wonder if there’s a point in maintaining a “sloppy” CHANGELOG_DEV.md if we already have a proper CHANGELOG.md. I’d probably put all of this there right away.

Reply:
`CHANGELOG.md`:
* contains summary for each release
* is actual for `stable` branch only
`CHANGELOG_DEV.md`:
* contains a bit more technical details
* something like communication between developers
* is actual for each branch where it present

Reply:
Almost everything you add here is going to be important for release notes too.

The developers could benefit from links to pull requests that we maintain in CHANGELOG.md. That makes it possible to see all the gory technical details if necessary.

I also believe it is more predictable to have a single CHANGELOG.md which is actual for the branch it is in. That kinda simplifies things. If I want to read a changelog for stable, I check out stable and read it there. If I want master version, I check out master. If I want to peek at some long-running development branch then I just go there. I still need to do this with CHANGELOG_DEV, but now I need to peek into a different file and disregard the regular CHANGELOG which just sits there being a pretty flower. I don’t know. It just looks as if we did all the development in a separate directory `src_dev` and then merged it into `src` during the release.